src/components/ActiveVideo/ActiveVideo.tsx

```diff
@@ -0,0 +1,57 @@
+import { ActiveVideoCapture, LivenessError } from '@onfido/active-video-capture'
+import { h, FunctionComponent } from 'preact'
+import { useState } from 'preact/hooks'
+import { localised } from '~locales'
+import type { WithLocalisedProps } from '~types/hocs'
+import type { StepComponentBaseProps } from '~types/routers'
+
+type Props = StepComponentBaseProps & WithLocalisedProps
+
+const ActiveVideo: FunctionComponent<Props> = (props) => {
+  const { nextStep } = props
+  const [videoPayload, setVideoPayload] = useState<Blob | null>(null)
```

Comment:
`videoPayload` is not used. Do you really need this state ?

Reply:
Yeah, that's the output of the step, it's essential 😅 We just don't upload it yet, but this MR is already big enough. It will be used in a follow-up MR.